File: samples/client/petstore/java/src/main/java/org/openapijsonschematools/client/components/securityschemes/apikeyquery.java
package org.openapijsonschematools.client.components.securityschemes;

import org.openapijsonschematools.client.securityschemes.ApiKeySecurityScheme;
import org.openapijsonschematools.client.securityschemes.ApiKeyInLocation;

public class ApiKeyQuery extends ApiKeySecurityScheme {
    /*
    apiKey in query
    */
    public ApiKeyQuery(String apiKey) {
        super(apiKey, "api_key_query", ApiKeyInLocation.QUERY);
    }
}

File: samples/client/petstore/kotlin/src/main/kotlin/org/openapijsonschematools/client/components/securityschemes/ApiKeyQuery.kt
package org.openapijsonschematools.client.components.securityschemes;

import org.openapijsonschematools.client.securityschemes.ApiKeyQuerySecurityScheme;

public class ApiKeyQuery extends ApiKeyQuerySecurityScheme {
    /*
    apiKey in query
    */
    public ApiKeyQuery(String apiKey) {
        super(apiKey, "api_key_query");
    }
}
